ballerina/client.bal

```diff
@@ -341,6 +341,194 @@ public isolated client class Client {
+    # + return - An error on failure or else `()`
+    remote isolated function UploadPart(
+            @display {label: "Object Name"} string objectName,
+            @display {label: "Bucket Name"} string bucketName,
+            @display {label: "File Content"} string|xml|json|byte[]|stream<io:Block, io:Error?> payload,
+            @display {label: "Upload ID"} string uploadId,
+            @display {label: "Part Number"} int partNumber,
+            @display {label: "UploadPartHeaders"} UploadPartHeaders? uploadPartHeaders = ()
+        ) returns CompletedPart|error {
+
+        if objectName == EMPTY_STRING {
+            return error(EMPTY_OBJECT_NAME_ERROR_MSG);
+        }
+        if bucketName == EMPTY_STRING {
+            return error(EMPTY_BUCKET_NAME_ERROR_MSG);
+        }
+        http:Request request = new;
+
+        string requestURI = string `/${bucketName}/${objectName}`;
+        string queryParamStr = string `?partNumber=${partNumber}&uploadId=${uploadId}`;
+        
+        map<string> queryParamsMap = {
+            "partNumber": partNumber.toString(),
+            "uploadId": uploadId
+        };
+
+        map<string> requestHeaders = setDefaultHeaders(self.amazonHost);
+
+        if uploadPartHeaders != () {
+            populateUploadPartHeaders(requestHeaders, uploadPartHeaders);
+        }
+
+        if payload is byte[] {
+            request.setBinaryPayload(payload);
+        } else if payload is stream<io:Block, io:Error?> {
+            request.setByteStream(payload);
+        } else {
+            request.setPayload(payload);
+        }
+
+        check generateSignature(self.accessKeyId, self.secretAccessKey, self.region, PUT, requestURI, UNSIGNED_PAYLOAD,
+            requestHeaders, request, queryParams = queryParamsMap);
+        requestURI = string `${requestURI}${queryParamStr}`;
+
+        http:Response httpResponse = check self.amazonS3->put(requestURI, request);
+        if httpResponse.statusCode == http:STATUS_OK {
+            string ETag = check httpResponse.getHeader("ETag");
+            return {partNumber, ETag};
+        } else {
+            xml XMLPayload = check httpResponse.getXmlPayload();
+            return error(XMLPayload.toString());
+        }
+    }
+
+    # Completes a multipart upload by assembling previously uploaded parts.
+    #
+    # + objectName - The name of the object  
+    # + bucketName - The name of the bucket
+    # + uploadId - The upload ID of the multipart upload  
+    # + CompletedParts - An array containing the part number and ETag of each uploaded part
+    # + return - An error on failure or else `()`
+    remote isolated function completeMultipartUpload(
+            @display {label: "Object Name"} string objectName,
+            @display {label: "Bucket Name"} string bucketName,
+            @display {label: "Upload ID"} string uploadId,
+            @display {label: "Array of Parts"} CompletedPart[] CompletedParts
+            ) returns @tainted error? {
```

Comment:
```suggestion
            ) returns error? {
```

Reply:
Why is tainted not used anymore?

Reply:
@nisuraaa Ballerina taint checking support was a removed sometimes ago and there's a plan to bring it back (ETA not decided). Therefore its recommended to avoid using the taint checking annotations, until we get back with the new support.

Reply:
Therefore please remove the related annotations in all the newly added code.